airflow/contrib/operators/sqoop_operator.py

```diff
@@ -25,8 +25,15 @@
 
 class SqoopOperator(BaseOperator):
     """
-    execute sqoop job
+    Execute a Sqoop job.
+    Documentation for Apache Sqoop can be found here: https://sqoop.apache.org/docs/1.4.2/SqoopUserGuide.html.
     """
+    template_fields = ('conn_id', 'cmd_type', 'table', 'query', 'target_dir', 'file_type', 'columns', 'split_by',
```

Comment:
making all the string fields templatable so can be seen in UI rendered template view and can pass in other templatable arguments like ds etc if need be